SCOPE.md

```diff
@@ -36,7 +36,7 @@ The table specifies whether the feature/component is in or out of scope.
 | Name | Description | In/Out | Reason |
 |------------------------------|--------------------------------------------------------------------------------------------------------|--------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | execution | Provide an extensible execution layer for executing a container | in | Create,start, stop pause, resume exec, signal, delete |
-| cow filesystem | Built in functionality for overlay, aufs, and other copy on write filesystems for containers | in |  |
+| cow filesystem | Built in functionality for overlay and other copy on write filesystems for containers | in |  |
```

Comment:
I assume "`For the scope of this project to change it requires a 100% vote from all maintainers of the project`" does not apply to this change.